test/integration/test-amp-story-analytics.js

```diff
@@ -0,0 +1,165 @@
+                "on": "story-bookend-enter",
+                "request": "endpoint",
+                "extraUrlParams": {
+                  "bookendEnter": true
+                }
+              },
+              "trackBookendExit": {
+                "on": "story-bookend-exit",
+                "request": "endpoint",
+                "extraUrlParams": {
+                  "bookendExit": true
+                }
+              }
+            },
+            "extraUrlParams": {
+              "pageVisible": "\${storyPageId}",
+              "bookendEnter": false,
+              "bookendExit": false,
+              "muted": false,
+              "unmuted": false
+            }
+          }
+          </script>
+        </amp-analytics>
+        <amp-story-page id="page-1">
+          <amp-story-grid-layer template="vertical">
+            <h1>First page</h1>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="page-2">
+          <amp-story-grid-layer template="vertical">
+            <h1>Second page</h1>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-bookend layout="nodisplay">
+          <script type="application/json">
+          {
+            "bookendVersion": "v1.0",
+            "shareProviders": [
+              {
+                "provider": "facebook",
+                "data-param-app_id": "1682114265451337",
+                "data-param-href": "https://fr-fr.facebook.com/LaRochePosayFrance/"
+              },
+              {
+                "provider": "twitter",
+                "data-param-url": "https://twitter.com/larocheposayfr?lang=fr"
+              }
+            ],
+            "components": [
+              {
+                "type": "heading",
+                "text": "Learn more about our 0% formulation charter"
+              },
+              {
+                "type": "cta-link",
+                "links": [
+                  {
+                    "text": "Click here",
+                    "url": "https://www.laroche-posay.fr/produits-soins/anthelios/peaux-sensibles-ou-allergiques-au-soleil-r93.aspx"
+                  }
+                ]
+              },
+              {
+                "type": "landscape",
+                "title": "TRAPPIST-1 Planets May Still Be Wet Enough for Life",
+                "url": "http://example.com/article.html",
+                "category": "astronomy",
+                "image": "http://placehold.it/360x760"
+              }
+            ]
+          }
+          </script>
+        </amp-story-bookend>
+      </amp-story>`;
+    describes.integration('amp-story analytics', {body, extensions}, env => {
+      let browser;
+      beforeEach(() => {
+        browser = new BrowserController(env.win);
+      });
+
+      it('should send analytics event when landing on a page', async () => {
+        await browser.waitForElementLayout('#page-1');
+        await RequestBank.withdraw().then(req => {
```

Comment:
Here and below, please only use async/await or promises, you don't need both.
`await` uses generators, so it'll wait until the promises resolves before going to the next line.
If you need to get the value returned by the promise: `const foo = await bar();`

Reply:
Got it, thanks. Done 👍